server/routes/imageRouter.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express'
+import csurf from 'csurf'
+
+const testMode = process.env.NODE_ENV === 'test'
+
+export default function imageRouter(): Router {
```

Comment:
Shouldn't this router still use the authentication router? I'm assuming only authenticated users can get images right?

(sorry if this is done somewhere else)

Reply:
Yes only auth user can get images, this still applies